sd: sdmmc: initialize cmd retries

jgenssler-GiN opened the pull request:
Sets the retries in cmd struct to their Kconfig value. This fixes an issue in `imx_usdhc.c` where retries would be used uninitialized. That leads to a loop not being entered and subsequently no data being read from the controller.
Tested on `mimxrt1170_evk`.

danieldegrasse commented on subsys/sd/sdmmc.c:
```diff
@@ -387,28 +395,27 @@ static int sdmmc_set_current_limit(struct sd_card *card)
 		max_current = SD_SET_CURRENT_600MA;
 	} else if (card->host_props.max_current_180 >= 400 &&
-		(card->switch_caps.sd_current_limit & SD_MAX_CURRENT_400MA)) {
+		   (card->switch_caps.sd_current_limit & SD_MAX_CURRENT_400MA)) {
```
Are you running `clang-format` on this file? If you'd like to do so you are welcome to, but please make a commit with *just* those changes. Putting both changes in one commit makes this type of otherwise simple fix difficult to review

jgenssler-GiN replied:
Sure, will do!

Commit:
sd: sdmmc: initialize cmd retries

Sets the retries in cmd struct to their Kconfig value. This fixes
an issue in imx_usdhc.c where retries would be used uninitialized. That
leads to a loop not being entered and subsequently no data being read
from the controller.
Tested on mimxrt1170_evk.

Signed-off-by: Jakob Genßler <jakob.genssler@gin.de>

Commit:
sd: sdmmc: fix formatting

Formats according to latest .clang-format rules.

Signed-off-by: Jakob Genßler <jakob.genssler@gin.de>

jgenssler-GiN commented:
Hi @danieldegrasse, the Compliance Checks seems to fail because I used the German spelling of my name for the commit author instead of the international spelling in the `Signed-off-by:` note.
Would you like me to push again with both matching? If so, can the check handle 'ß' or should I use 'ss' instead?

aasinclair commented:
> Hi @danieldegrasse, the Compliance Checks seems to fail because I used the German spelling of my name for the commit author instead of the international spelling in the `Signed-off-by:` note. Would you like me to push again with both matching? If so, can the check handle 'ß' or should I use 'ss' instead?

ß should be OK in the author name, it just needs the line to match, so this should work:
`Signed-off-by: Jakob Genßler <jakob.genssler@gin.de>`
